perf/src/main/scala/cromwell/perf/CompareMetadata.scala

```diff
@@ -18,14 +21,35 @@ import io.circe.java8.time.decodeOffsetDateTimeDefault
 object CompareMetadata extends App with StrictLogging{
   private  val REGRESSION_CONST = 1.1
 
-  def parseMetadataFromFile(filePath: String): Either[circe.Error, Metadata] = {
+  def parseMetadataFromLocalFile(filePath: String): Either[circe.Error, Metadata] = {
     val metadataFile = File(filePath)
     val metadataFileContent = metadataFile.contentAsString
 
     decode[Metadata](metadataFileContent)
   }
 
 
+  def parseMetadataFromGcsFile(gcsUrl: String, pathToServiceAccount: String): Either[circe.Error, Metadata] = {
+    val gcsUrlArray = gcsUrl.replace("gs://", "").split("/", 2)
+    val fileToBeLocalized = gcsUrlArray(1)
+    val gcsBucket = gcsUrlArray(0)
+
+    val credentials = GoogleCredentials.fromStream(new FileInputStream(pathToServiceAccount))
+    val storage = StorageOptions.newBuilder().setCredentials(credentials).build().getService
+    val blob = storage.get(gcsBucket, fileToBeLocalized)
+    val metadataFileContent = new String(blob.getContent())
```

Comment:
ToL:

<img src="https://user-images.githubusercontent.com/791985/46876090-69376400-ce0b-11e8-9b8f-49ef6df82c6e.png" alt="woodard and cromwell" title="woodard and cromwell" width="20%">

Someday...

Reply:
I changed `new String` to more Scala-y way now